### DISCUSSION TITLE
[Feature Request] Server Components by Default

### DISCUSSION THREAD

**Discussion by ghost** [heroui-inc/heroui#979] [Announcements]
### Is your feature request related to a problem? Please describe.

I have to do "use client" in a NextJS 13 app with experimental AppDir feature.

### Describe the solution you'd like

I expected it works on the new AppDir feature, but not in the current version.

### Describe alternatives you've considered

.

**Comment by jrgarciadev** [comment]
Hey @nluziaf this doesn't depend on the NextUI team, it depends on the `stitches` team, since NextUI is based on `Stitches`, I'm afraid they will not add support for static extraction due to it being quite challenging to achieve specifically by the dynamic variants, it would generate thousands of lines of CSS code, here's an example:https://vanilla-extract.style/documentation/packages/sprinkles/

You can follow up on the future of Stitches here https://github.com/stitchesjs/stitches/discussions/1053#discussioncomment-3078853, but at the moment we only support SSR in the `pages` NextJs folder, otherwise, you need to use `use client`, on the other hand the `app` feature is still experimental by NextJS team so you shouldn't create an issue on any of the `CSS-in-JS` UI libraries 👍🏻

**Comment by wangly19** [comment]
`AppDir` is currently in Beta stage, `NextUI` uses `Stitches` to create a design system. The `Stitches` team should have received some feedback on `Next13`.
I believe all issues will be resolved in the near future, but for now it seems that the developers of css in js are waiting for a point in time.

The official document makes a statement,  [Click](https://beta.nextjs.org/docs/styling/css-in-js)
<img width="2603" alt="image" src="https://user-images.githubusercontent.com/45115006/211004679-8f7a219b-bbb8-4e23-9207-75aa02adf9f0.png">
